app/views/station/new.erb

```diff
@@ -4,31 +4,40 @@
 </head>
 
 <div class="transbox">
-  <form class="form-horizontal">
+  <form class="form-horizontal" action="/stations" method="post">
     <fieldset>
+      <legend>New Id</legend>
```

Comment:
Is it part of the requirement that you specify what the id is of the new station?